thanks.html

```diff
@@ -115,6 +115,62 @@
 
 
 	</style>
+<script>
+function insertUrlParam(key, value) {
+	if (history.pushState) {
+		let searchParams = new URLSearchParams(window.location.search);
+		searchParams.set(key, value);
+		let newurl = window.location.protocol + "//" + window.location.host + window.location.pathname + '?' + searchParams.toString();
+		window.history.pushState({path: newurl}, '', newurl);
+	}
+}
+
+const params = new Proxy(new URLSearchParams(window.location.search), {
+  get: (searchParams, prop) => searchParams.get(prop),
+});
+
+let recorded = params.success; 
+
+var usecase = '';
+{% if action.page.custom_fields.page_use_case %} usecase = '{{ action.page.custom_fields.page_use_case }}';{% endif %}

```

Comment:
FYI `usecase` is an optionally provided custom page field, which if provided, will be recorded under the `affiliation` property of the transaction event